io.github.lunarequest.NightPDF.yaml

```diff
@@ -0,0 +1,59 @@
+app-id: io.github.lunarequest.NightPDF
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```

Comment:
can 22.08 be used?

Reply:
it should

Reply:
this is not addresser here. Version shoud match in both places.